src/ansys/sound/core/psychoacoustics/spectral_centroid.py

```diff
@@ -134,9 +139,9 @@ def __str__(self):
         str
             Formatted info about self.
         """
-        str = __class__.__name__ + "\n"
+        str = __class__.__name__ + " object\n"
         str += "Data\n"
         str += "\t Signal name: " + self.signal.name + "\n"
-        str += f"\t Spectral centroid: {self.get_spectral_centroid()} Hz\n"
+        str += f"\t Spectral centroid: {self.get_spectral_centroid():.1f} Hz\n"
```

Comment:
@anshlachamb unless it has changed, you will run into problems with some pre-commit tools, as it enforces a whitespace after each colon character. To circumvent that (if it is still an issue), I used np.round() (or you can also add the white space here and remove that which is before the '{' to compensate).

Reply:
(Just ignore this comment if all checks are passed...)

Reply:
[not necessary]